GlobalSuppressions.cs

```diff
@@ -0,0 +1,8 @@
+﻿// This file is used by Code Analysis to maintain SuppressMessage
+// attributes that are applied to this project.
+// Project-level suppressions either have no target or are given
+// a specific target and scoped to a namespace, type, member, etc.
+
+using System.Diagnostics.CodeAnalysis;
+
+[assembly: SuppressMessage("Performance", "CA1822:Mark members as static", Justification = "<Pending>", Scope = "member", Target = "~M:TotalImage.Validation.ValidationMethods.FirstCharacterValidation.ValidationCore(System.Object)~System.Object")]
```

Comment:
I'm not sure there's any good reason to have this member not marked as static? See below.